Makefile

```diff
@@ -164,3 +164,23 @@ name-collision:
 
 composer-dependency-analyser:
 	php vendor/bin/composer-dependency-analyser --config build/composer-dependency-analyser.php
+
+.PHONY: infection
+infection:
+	git -C build-infection pull || git clone https://github.com/phpstan/build-infection.git
+	git -C build-infection fetch origin && git -C build-infection reset --hard origin/1.x
+	composer install --working-dir build-infection --no-interaction --no-progress
+	php build-infection/bin/infection-config.php --source-directory='build/PHPStan/Build'> infection.json5
+	XDEBUG_MODE=coverage php tests/vendor/bin/paratest \
+		--coverage-xml=tmp/coverage/coverage-xml \
+		--log-junit=tmp/coverage/junit.xml
+	version=$(shell jq -r '.packages[] | select(.name == "infection/infection") | .version' build-infection/composer.lock); \
+		wget https://github.com/infection/infection/releases/download/$$version/infection.phar --timestamping;
+	php infection.phar \
+		--configuration=infection.json5 \
+		--git-diff-base=origin/2.1.x \
```

Comment:
`origin/HEAD` would also work and wouldn't have to be updated when the branch changes?

Reply:
fixed. thanks